### PR TITLE
Add quick steps for adding docs to cmdlets

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
# PR Summary

These are the quick steps for using PlatyPS to update docs for you.

## PR Context

I've had to ask a couple of times so I'm documenting it so it's easy to link to contributors.
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -41,6 +41,32 @@ Please read the rest of this document to ensure a smooth contribution process.
 
 Please see the [Contributor Guide in `MicrosoftDocs/PowerShell-Docs`](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md).
 
+#### Quick steps if you're changing an existing cmdlet
+
+If you made a change to an existing cmdlet and would like to update the documentation using PlatyPS,
+here are the quick steps:
+
+1. Install 
+`PlatyPS`
```
Comment (TravisEz13) [inline .github/CONTRIBUTING.md]: Make sure you use markdownlint and correct all issue

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comments

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (TylerLeonhardt) [reply .github/CONTRIBUTING.md]: done.

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
@TravisEz13 Please update your review.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**